src/importmesh.cpp

```diff
@@ -85,7 +108,8 @@ static hEntity newLine(EntityList *el, int id, hEntity p0, hEntity p1) {
     en.actVisible = true;
     en.forceHidden = false;
 
-    en.h.v = id + en.group.v*65536;
+    en.h.v = *id + en.group.v*65536;
+    *id = *id + 1;
```

Comment:
This replaces the `id++` in the calls below. OK.

Reply:
But why did you decide to do it? Just aesthetics or am I missing something?

Reply:
This was borrowed from the way it's done in idfimport. Over there I did it because that's how I saw them combined somewhere else? Using a pointer to the id was also borrowed from the idf code because I'd like to eventually use the same set of entity creation functions in both. The ID stuff is still strange to me overall, so I'm not real sure about it.